Corona-Warn-App/src/main/res/layout/bugreporting_debuglog_fragment.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <Toolbar
+        android:id="@+id/toolbar"
+        style="@style/CWAToolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:navigationIcon="@drawable/ic_back"
+        android:title="@string/debugging_debuglog_title" />
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+            <TextView
+                android:id="@+id/explanation"
+                style="@style/body1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/spacing_normal"
+                android:text="@string/debugging_debuglog_intro_explanation" />
+
+            <TextView
+                android:id="@+id/sensitive_information"
+                style="@style/body1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@color/cwaGrayHighlight"
+                android:padding="@dimen/spacing_normal"
+                android:text="@string/debugging_debuglog_intro_warning"
+                android:textColor="@color/colorStableLight" />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                style="@style/cardTracing"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/spacing_small"
+                android:layout_marginTop="@dimen/spacing_normal"
+                android:layout_marginEnd="@dimen/spacing_small"
+                android:layout_marginBottom="@dimen/spacing_tiny">
+
+                <ProgressBar
+                    android:id="@+id/debuglog_activity_indicator"
+                    android:layout_width="36dp"
+                    android:layout_height="36dp"
+                    android:visibility="gone"
+                    app:layout_constraintBottom_toBottomOf="@+id/debuglog_status_secondary"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="@+id/debuglog_status_primary"
+                    tools:visibility="visible" />
+
+                <TextView
+                    android:id="@+id/debuglog_status_primary"
+                    style="@style/body1"
+                    android:layout_width="0dp"
```

Comment:
Some layouts use `@dimen/match_constraint` here. Is this still a thing? Then it should be used here and for the following views too.

Reply:
While we are still discussing use of `spacing_*`, I'm all for removing `@dimen/match_constraint`.
Neither plays well with the constraint layout editor.

I'll follow up with a PR that removes `@dimen/match_constraint` from the project.